contracts/Manager.sol

```diff
@@ -761,10 +991,14 @@ contract Manager is UUPSOwnableUpgradeable, UsingRegistryUpgradeable {
      * @param group The group to deprecate.
      */
     function _deprecateGroup(address group) private {
-        if (!activeGroups.remove(group)) {
-            revert GroupNotActive(group);
+        bool activeGroupsRemoval = activeGroups.remove(group);
+        bool specificGroupsRemoval = specificStrategyVotedGroups.remove(group);
```

Comment:
I'm wondering if we should have a separate mechanism for  blocking a group from being used in a specific strategy. We might want to deprecate a group from the default strategy, but still allow it to be voted on with the specific strategy (which I think the current design doesn't support, right?).

Reply:
I updated the implementation and currently it is possible to deprecated group and block strategy completely independently